Makefile

```diff
@@ -54,17 +62,11 @@ clear-caches:
 
 .PHONY: setup
 setup: .env
-	docker-compose run web /app/bin/setup-services.sh
+	docker-compose run --rm web /app/bin/setup-services.sh
 
 .PHONY: shell
 shell: .env .docker-build
-	# Use `-u 0` to automatically become root in the shell
-	docker-compose run --user 0 web bash
-
-.PHONY: currentshell
-currentshell: .env .docker-build
-	# Use `-u 0` to automatically become root in the shell
-	docker-compose exec --user 0 web bash
+	docker-compose run --rm web bash
```

Comment:
This was leaking docker containers. This should reduce the number of containers created and orphaned.